packages/atomic/src/components/facets-v1/facet-common.ts

```diff
@@ -21,12 +21,6 @@ export interface FacetValueProps {
   searchQuery?: string;
 }
 
-export interface FacetValueLabelHighlightProps {
-  displayValue: string;
-  searchQuery?: string;
-  isSelected: boolean;
-}
```

Comment:
Is this being moved from `facet-common.ts` to the file that directly uses it because no other files are planning on importing it and using this interface?

Reply:
Yes it makes sense that the FacetValueLabelHighlightProps go along with the FacetValueLabelHighlight component, it's not common to all facets